### PR TITLE
Extend marmoset to provide callbackup endpoint for installimage issue62

### PR DESCRIPTION
**Pull request by bastelfreak**
Do not merge!

this PR is currently only used to provide a shortcut to the diff.

### CODE REVIEW

**Comment by bastelfreak** [comment]
this will implement https://github.com/virtapi/marmoset/issues/62

File: marmoset.conf.example [hunk 1]
```diff
@@ -66,3 +67,6 @@ ldap_client_base_dn = cn=static,cn=DHCP Service Config,dc=example,dc=com
 
 # No empty gateway are allowed
 force_gateway: True
+
+[Installstatus]
+SQLiteDB: /opt/marmoset/installstatus.db
```
Comment (bastelfreak) [inline marmoset.conf.example]: is it possible to move this to a relative path?

Comment (pbyatshon) [inline marmoset.conf.example]: sure...

File: marmoset.conf.example [hunk 2]
```diff
@@ -8,6 +8,7 @@ VM: True
 Webserver: True
 Installimage: True
 DHCP: True
+Installimage: True
```
Comment (bastelfreak) [inline marmoset.conf.example]: this is a duplicate with line 9 and should be Installstatus?

Comment (pbyatshon) [inline marmoset.conf.example]: yes, that should have been Installstatus. corrected now.

**Comment by bastelfreak** [comment]
@pbyatshon any opinions here? In my opinion we can merge this.

**Comment by pbyatshon** [comment]
how about adding a status code in addition to the status message? I guess that would be better than including it in the status message itself.

**Comment by bastelfreak** [comment]
We tested this, works fine right now, except for one typo in a return statement + the wrong path. Message will be split up into several steps
- message
- exitcode
- current step
- amount of steps

**Comment by bastelfreak** [comment]
tested an works

**Pull request merged**